### PR TITLE
added the first block of the home page

### PR DESCRIPTION
**Pull request by Cra7yy**
added the first block of the main page has configured the adaptivity so far works as in the old project

### CODE REVIEW

**Comment by Cra7yy** [comment]
fix object

File: src/components/creatorNetwork/creator-network.jsx [hunk 1]
```diff
@@ -1,72 +1,84 @@
 import {useState, useEffect} from 'react'
 import {StyledCreatorNetwork} from './creator-network.styled'
 import {Container} from '../../globalStyled/styled.global'
-import {Timer} from './timer/timer'
+import {VicupPrice} from "./vicupPrice/vicup-price"
+import {Course} from "./course/course";
```
Comment (Capsonchik) [inline src/components/creatorNetwork/creator-network.jsx]: codestyle

File: src/components/creatorNetwork/creator-network.jsx [hunk 2]
```diff
@@ -1,72 +1,84 @@
...
+
+  console.log('auctionCard', auctionCard)
+  // console.log('auctionCard', auctionCard.user)
+  // console.log('auctionCard', auctionCard.card)
+  console.log('newArrAuction', newArrAuction)
 
-  setDataUserAuction()
-  setDataCardAuction()
 
   const perecklPlus = () => {
```
Comment (Capsonchik) [inline src/components/creatorNetwork/creator-network.jsx]: stepUp

File: src/components/creatorNetwork/creator-network.jsx [hunk 3]
```diff
@@ -1,72 +1,84 @@
...
-      setIndex( 0)
-    }else{
+    if (index === newArrAuction.length - 1) {
+      setIndex(0)
+    } else {
       setIndex(index + 1)
     }
   }
 
   const precklMinus = () => {
```
Comment (Capsonchik) [inline src/components/creatorNetwork/creator-network.jsx]: stepDown

File: src/components/creatorNetwork/creator-network.jsx [hunk 4]
```diff
@@ -1,72 +1,84 @@
...
-      setIndex(newArrAuction.length -1)
-    }else{
+    if (index === 0) {
+      setIndex(newArrAuction.length - 1)
+    } else {
       setIndex(index - 1)
     }
   }
 
-  const [ vicupPrice, setVicupPrice] = useState('')
```
Comment (Capsonchik) [inline src/components/creatorNetwork/creator-network.jsx]: currentPrice

**Review by Capsonchik — CHANGES_REQUESTED** [review]
Хуйня переделывай =)))

**Pull request merged**